package.json

```diff
@@ -60,7 +63,8 @@
     "bill": "^2.2.1",
     "dom-helpers": "^2.4.0",
     "invariant": "^2.2.0",
-    "lodash": "^3.10.1",
+    "lodash.create": "^4.0.3",
```

Comment:
I actually prefer to use the normal lodash as its easier to opt into methods but maintains the same benefit of only including code you use